bop_toolkit_lib/dataset/bop_v2.py

```diff
@@ -0,0 +1,260 @@
+import pathlib
```

Comment:
maybe adding some comments about the formats, like the data structure, definitions, etc.? That would help users understand the formats more easily.

Reply:
Documentation has been added to the top of the files